### PR TITLE
[BTFS-1706] Hub 2.0

### PR DESCRIPTION
**Pull request by wxue**

### CODE REVIEW

File: protos/hub/hub.proto [hunk 1]
```diff
@@ -97,6 +99,42 @@ message StatsResp {
...
+  float download_average = 5;
+  float download_variance = 6;
+}
+
+message ParseReq {
+  repeated Candidate candidates = 1;
+}
+
+message RecycleReq {
+  string index_name = 1;
```
Comment (taiyangc) [inline protos/hub/hub.proto:124]: where is this from? hard to see where this string can be obtained

Comment (taiyangc) [reply protos/hub/hub.proto:124]: some kind of id or somewhere from response is better for understanding

Comment (wxue) [reply protos/hub/hub.proto:124]: this is to specify the ES index by its name that need to be recycled

Comment (wxue) [reply protos/hub/hub.proto:124]: will change to enum per discussion

**Pull request merged**